newtab.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Bespoke Chrome Dashboard Ver. 10</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="">
+        <script>
+            function myFunction() {
+              document.getElementById("demo").src = "https://source.unsplash.com/random/1440x900";
+            }
+            </script>
+              <link rel="stylesheet" href="./style.css">
```

Comment:
It is best practice to exercise DRY (Don't Repeat Yourself) code...

This is a duplicate (albeit complete in this version) line of code for that which is stated above. Is this due to the use of a newtab.html file from the Chrome Extension Documentation?

Additionally, given that this is a relative link, it does not require the preceding ./ to link to the CSS document.